notebooks/working/6.0-mab-estimate-causal-effect.md

```diff
@@ -0,0 +1,2197 @@
+                                                  np.random.normal(loc=0,
+                                                                   scale = WTW_Reg['total_travel_cost_on_total_travel_time'].resid.std(),
+                                                                   size = long_sim_data_causal.loc[long_sim_data_causal['mode_id']==4].shape[0]))
+```
+
+### Drive-Transit-Walk
+
+```python
+long_sim_data_causal.loc[long_sim_data_causal['mode_id']==5,'total_travel_cost'] = \
+                                                 (DTW_Reg['total_travel_cost_on_total_travel_time'].params[0] +\
+                                                  DTW_Reg['total_travel_cost_on_total_travel_time'].params[1] *\
+                                                  long_sim_data_causal.loc[long_sim_data_causal['mode_id']==5,'total_travel_time'] +\
+                                                  np.random.normal(loc=0,
+                                                                   scale = DTW_Reg['total_travel_cost_on_total_travel_time'].resid.std(),
+                                                                   size = long_sim_data_causal.loc[long_sim_data_causal['mode_id']==5].shape[0]))
+```
+
+### Walk-Transit-Drive
+
+```python
+long_sim_data_causal.loc[long_sim_data_causal['mode_id']==6,'total_travel_cost'] = \
+                                                 (WTD_Reg['total_travel_cost_on_total_travel_time'].params[0] +\
+                                                  WTD_Reg['total_travel_cost_on_total_travel_time'].params[1] *\
+                                                  long_sim_data_causal.loc[long_sim_data_causal['mode_id']==6,'total_travel_time'] +\
+                                                  np.random.normal(loc=0,
+                                                                   scale = WTD_Reg['total_travel_cost_on_total_travel_time'].resid.std(),
+                                                                   size = long_sim_data_causal.loc[long_sim_data_causal['mode_id']==6].shape[0]))
+```
+
+### Walk 
+
+
+No further change of variables since travel distance directly affects the utility.
+
+
+### Bike 
+
+
+No further change of variables since travel distance directly affects the utility.
+
+```python
+estimated_probabilities = mnl_model_sim.predict(long_sim_data_causal)
+```
+
+```python
+long_sim_data_causal['estimated_probabilities'] = estimated_probabilities
+```
+
+## Based on True Causal Graph and Original Estimated Model
+
+```python
+true_probabilities = mnl_model.predict(long_sim_data_causal)
+```
+
+```python
+long_sim_data_causal['true_probabilities'] = true_probabilities
+```
+
+### Plots
+
+```python
+long_sim_data['initial_probabilities'] = initial_probabilities
+```
+
+```python
+naive_effect = long_sim_data_naive.loc[long_sim_data_naive['mode_id']==1,'naive_probabilities'] - long_sim_data.loc[long_sim_data['mode_id']==1,'initial_probabilities']
```

Comment:
Here, you're looking at the `naive_effect` as a vector, as in 1 per observation. However, we want `naive_effect` to be once per simulation so `naive_effect = naive_effect.mean)`.

The same goes for your `estimated_effect` and `true_effect`.

Reply:
Also, per the last comment in #74 can you use the following causal effect estimate for the naive, estimated, and true estimates?
```suggestion
naive_effect = long_sim_data_naive.loc[long_sim_data_naive['mode_id'].isin([1, 2, 3]), 'naive_probabilities'] - long_sim_data.loc[long_sim_data['mode_id'].isin([1, 2, 3]), 'initial_probabilities']
```

Reply:
no worries!